src/frontend/utils/parseDataElements/parseDataElements.spec.ts

```diff
@@ -35,9 +38,19 @@ describe.only('utils/parseDataElements', () => {
       };
       dataElement.setAttribute('data-context', JSON.stringify(context));
 
-      expect(() => {
-        parseDataElements(dataElement);
-      }).toThrowError(`Model ${modelName.THUMBNAILS} not supported`);
+      let data: AppData;
+      let thrownError;
+      try {
+        data = parseDataElements(dataElement);
+      } catch (error) {
+        thrownError = error;
+      }
```

Comment:
```suggestion
      const data = parseDataElements(dataElement);
```

The `try/catch` here is useless. `parseDataElements` must not throw an error anymore. By removing the try/catch you ensure that it never happen. If it happens, the your test will fail !